start.py

```diff
@@ -55,7 +56,17 @@ def main(simulation_parameters):
             allagents.do('filobl')
             insurancecustomers.do('check_risk')
 
-        simulation.graphs()
+        if not isTravis:
+            simulation.graphs()
```

Comment:
@x0range, Boolean `isTravis` is wired from the `parser.add_argument` with a default to be `False`. I think in these cases, a default argument is cleaner to use than try-except. C struct and Protobuf have default values, for instance. Python dictionary has `.get` method with default value.